datamanagement/server_cleanup.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+import logging
+import json
+from dbclients.tantalus import TantalusApi
+from datamanagement.utils.runtime_args import parse_runtime_args
+
+if __name__ == "__main__":
+	logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
+	tantalus_api = TantalusApi()
+	args = parse_runtime_args()
+	data_to_check_list = []
+
+	shahlab_prefix = "/shahlab/archive"
+
+	blob_storage = tantalus_api.get_storage_client('singlecellblob')
+	shahlab_storage = tantalus_api.get_storage_client('shahlab')
+
+	#Change in future to accept command line args for different dataset types
+	all_bam_files = tantalus_api.list('sequence_dataset', dataset_type="BAM")
+
+	#Currently Tantalus doesn't have the library_type implemented, so have to filter via iteration
+	#Will change once Andrew implements the new filter
+	for dataset in all_bam_files:
+		if(dataset['library']["library_type"] == "SC_WGS" or dataset['library']['library_type'] == "WGS"):
+			data_to_check_list.append(dataset)
+		
+	logging.info(len(data_to_check_list))
+
+	total_data_size = 0
+
+	for data in data_to_check_list:
+		for file_resource_id in data['file_resources']:
```

Comment:
Same as for check_integrety, use tantalus_api.get_sequence_dataset_file_instances(dataset, storage_name)